src/sql/workbench/contrib/queryplan2/browser/queryPlan.ts

```diff
@@ -281,7 +283,14 @@ export class QueryPlan2 implements ISashLayoutProvider {
 	}
 
 	private populate(node: InternalExecutionPlanNode, diagramNode: any): any {
-		diagramNode.label = node.name;
+		diagramNode.label = '';
+		node.subtext.forEach((str, index) => {
```

Comment:
There is a join function in js for this. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join